docs/release-notes/long-metrics.rst

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**Bug Fixes**
+
+-  Experiment metrics tracking: add support for metrics with long names. Previously, a metric with a
+   name over 63 characters long will be recorded, but will not be displayed in the UI or returned by
+   the APIs.
```

Comment:
:orphan:

**Bug Fixes**

-  Experiment metrics tracking: Add support for metrics with long names. Previously, metrics with names exceeding 63 characters were recorded but not displayed in the UI or returned via APIs.